File: 2011/j3.py
# code by savir singh

t1 = int(input())
t2 = int(input())
sumacfound = 0
sumaclist = []
sumaclist.append(t1)
sumaclist.append(t2)

while sumacfound == 0:
    a = sumaclist[-2] - sumaclist[-1]
    if sumaclist[-1] < a:
        sumacfound = 1
    sumaclist.append(a)

print(len(sumaclist))

File: 2011/j3.cpp
// code by savir singh

#include <bits/stdc++.h>
#define int long long
#define del erase
using namespace std;

int32_t main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int num1, num2;
    cin >> num1 >> num2;
    vector<int> sumac = {num1, num2};
    sumac.push_back(num1-num2);
    if (abs(num1-num2)<num2) {
        while (true) {
            int num3 = abs(sumac[sumac.size()-1]-sumac[sumac.size()-2]);
            sumac.push_back(num3);
            if (num3>sumac[sumac.size()-2]) {
                break;
            }
        }
    }
    cout << sumac.size();
}
